server/database/index.js

```diff
@@ -0,0 +1,30 @@
+const mongoose = require('mongoose');
+
+const dbConnection = require('./dbConnection');
+const { createCollection, resetDatabase } = require('./utils');
+const { admin, client } = require('./data');
```

Comment:
I think you should wait for the rest of the table and insert them in this function